swap_meet/item.py

```diff
@@ -1,2 +1,30 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0.0,):
+        self.category = category
+        self.condition = condition
+
+    # Doing this made all the tests fail due to missing arguments, but
+    # To add the age attribute, I think that I would:
+    # def __init__(self, age, category = "", condition = 0.0,):
+        # self.age = age
+        # self.category = category
+        # self.condition = condition
+
+    # and then for each item category, I would change to:
+    # def __init__(self, age, category="Electronics", condition=0.0):
+    #     super().__init__(age, category, condition)
```

Comment:
We could add age in a similar fashion to the child classes. Again, we wouldn't want to disturb the existing parameter order, and we'd want to supply a default value.
```python
    def __init__(self, condition=0.0, age=0):
        super().__init__("Electronics", condition, age)
```